mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -28,7 +28,7 @@
 class KiViBufferWalker():
     """! Simple auxiliary class used to walk through a buffer and search for KV tokens """
     def __init__(self):
-        self.KIVI_REGEX = r"\{\{([\w\d_-]+);([^\}]+)\}\}\n"
```

Comment:
Yes. It can be removed from the regex. But used after receiving characters to see if one line is received. To satisfy DAPlink limitation.